src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleErrorFieldSymbol.vb

```diff
@@ -0,0 +1,100 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Immutable
+Imports System.Globalization
+Imports System.Threading
+Imports Microsoft.CodeAnalysis
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
+    ''' <summary>
+    ''' Represents a field of a tuple type (such as (int, byte).Item1)
+    ''' that doesn't have a corresponding backing field within the tuple underlying type.
+    ''' Created in response to an error condition.
+    ''' </summary>
+    Friend Class TupleErrorFieldSymbol
+        Inherits SynthesizedFieldSymbol
+
+        ''' <summary>
+        ''' If this field represents a tuple element (including the name match), 
+        ''' id is an index of the element (zero-based).
+        ''' Otherwise, (-1 - [index in members array]);
+        ''' </summary>
+        Private _tupleFieldId As Integer
+
+        Private _locations As ImmutableArray(Of Location)
+
+        Private _useSiteDiagnosticInfo As DiagnosticInfo
+
+        Public Overrides ReadOnly Property IsTupleField() As Boolean
+            Get
+                Return True
+            End Get
+        End Property
+
+        ''' <summary>
+        ''' If this field represents a tuple element (including the name match), 
+        ''' id is an index of the element (zero-based).
+        ''' Otherwise, (-1 - [index in members array]);
+        ''' </summary>
+        Public ReadOnly Property TupleFieldId() As Integer
+            Get
+                Return Me._tupleFieldId
+            End Get
+        End Property
+
+        Public Overrides ReadOnly Property TupleUnderlyingField() As FieldSymbol
+            Get
+                Return Nothing
+            End Get
+        End Property
+
+        Public Overrides ReadOnly Property Locations() As ImmutableArray(Of Location)
+            Get
+                Return Me._locations
+            End Get
+        End Property
+
+        Public Overrides ReadOnly Property DeclaringSyntaxReferences() As ImmutableArray(Of SyntaxReference)
+            Get
+                Return Symbol.GetDeclaringSyntaxReferenceHelper(Of VisualBasicSyntaxNode)(Me._locations)
+            End Get
+        End Property
+
+        Public Overrides ReadOnly Property IsImplicitlyDeclared() As Boolean
+            Get
+                Return False
+            End Get
+        End Property
+
+        Public Sub New(container As NamedTypeSymbol, name As String, tupleFieldId As Integer, location As Location, type As TypeSymbol, useSiteDiagnosticInfo As DiagnosticInfo)
+            MyBase.New(container, container, type, name)
+            Debug.Assert(name <> Nothing)
+            Me._locations = (If((location Is Nothing), ImmutableArray(Of Location).Empty, ImmutableArray.Create(Of Location)(location)))
```

Comment:
Fixed

---
In reply to: [71448051](https://github.com/dotnet/roslyn/pull/12596#discussion_r71448051) [](ancestors = 71448051)